doc/src/tutorial/index.rst

```diff
@@ -17,3 +17,4 @@
    solvers.rst
    matrices.rst
    manipulation.rst
+   codegen.rst
```

Comment:
I am -0.5 to including this in the tutorial. The tutorial is designed for beginners to SymPy, but this document is intended for users who are already familiar with the basics of SymPy. I would rather see this elsewhere (maybe modules, or special_topics).